Sources/Swift/Protocol/SentryLog.swift

```diff
@@ -17,42 +19,45 @@ struct SentryLog: Codable {
     
     /// The traceId is initially an empty default value and is populated during processing;
     /// by the time processing completes, it is guaranteed to be a valid non-empty trace id.
-    init(
+    public init(
```

Comment:
`m`: Why is this now public but hasn't been before? If it is public, and accessible from outside of the SDK we need a proper documentation

Reply:
Added documentation and also changed the init to internal, as users will mainly interact with the log through the callback and logger, and not create them directly.